### DISCUSSION TITLE
Start container synoai failed: {"message":"failed to initialize logging driver: database is locked"}.

### DISCUSSION THREAD

**Discussion by Neogx** [djdd87/SynoAI#120] [Q&A] [answered]
Everything was working perfectly then it just stopped working im not sure what that error is about. when it starts it right away turns off

**Comment by djdd87** [comment, marked as answer]
Synology Docker issue. Try rebooting your NAS. If that doesn't work, recreate the container. https://techoverflow.net/2021/03/23/how-to-fix-synology-docker-failed-to-initialize-logging-driver-database-is-locked/
